go.mod

```diff
@@ -54,3 +54,6 @@ require (
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+// FIXME: update after PR merged: https://github.com/bugsnag/panicwrap/pull/23
```

Comment:
can you create an issue for this?

Reply:
https://github.com/tharsis/ethermint/issues/311